src/vl-map-layer.js

```diff
@@ -238,6 +238,30 @@ export class VlMapLayer extends vlElement(HTMLElement) {
     }
   }
 
+  /**
+   * Verwijdert alle features van de laag
+   */
+  clearFeatures() {
+    if (this._source) {
+      this._source.clear();
+      this.__autoZoomToExtent();
+      this.rerender();
+    }
+  }
+
+  /**
+   * Voegt een feature toe aan de kaartlaag via geojson
+   *
+   * @param {string} feature
+   */
+  addFeature(feature) {
+    if (this._source) {
+      this._source.addFeatures([this._geoJSON.readFeature(feature)]);
```

Comment:
Dan kunnen we dan misschien ook beter ineens `addFeatures()` voorzien, waar `addFeature()` gebruik van kan maken? Zal efficienter zijn als je meerdere features tegelijk wilt adden.

Reply:
Stond niet in het ticket, maar kan uiteraard. Ik zal die rap toevoegen dan.